main.go

```diff
@@ -23,17 +25,35 @@ func printUsage() {
 	os.Exit(0)
 }
 
+func timeoutSecondsFromCount(count int) time.Duration {
+	waitOnLastOne := time.Duration(2) * time.Second
```

Comment:
```suggestion
	waitOnLastOne := 2 * time.Second
```

You don't need the cast here